src/app/components/MainContent/index.jsx

```diff
@@ -1,29 +1,19 @@
 import React from 'react';
-import styled from 'styled-components';
 import { any, arrayOf, shape } from 'prop-types';
-import { C_EBON, FF_NEWS_SANS_REG } from '../../../lib/constants/styles';
-
-const Headline = styled.h1`
-  color: ${C_EBON};
-  font-family: ${FF_NEWS_SANS_REG};
-  font-size: 2em;
-`;
+import Headline from '../Headline';
+import Text from '../Text';
 
 const render = ({ blocks }) =>
   blocks.map(block => {
     const { type, blockId } = block;
-    const blockString = JSON.stringify(block);
 
-    let Element = 'p';
+    let Element = Text;
```

Comment:
The genericised function in the branch linked to in the issue: https://github.com/bbc/simorgh/blob/recursive-model/src/app/optimo/renderBlocks.jsx seems much more future proof as a block of code.